README.md

```diff
@@ -1,2 +1,36 @@
 # gh-dependabot
-A GH CLI extension for Dependabot
+A GH CLI extension for using Dependabot
+
+## Installation
+
+```bash
+gh extension install https://github.com/therealkujo/gh-dependabot
+```
+
+## Usage
+
+### Export
+
+Exports all the dependabot alerts for a given repo(s) to a csv file
+
+```bash
+Usage: gh dependabot export [OPTIONS] [REPO]...
+
+  Pulls all dependabot alerts and exports them to a CSV file
+
+  REPO is space separated in the OWNER/NAME format
+
+Options:
+  -o, --output TEXT  Path to the output file  [required]
```

Comment:
How do you feel about making this optional using a generated name or output to stdout?

Reply:
Great point. I can make that optional and to just set the default behaviour to output to stdout instead

Reply:
Done! I changed the default behavior to just print to stdout and then the flag if you want to save it to a file